src/components/renderers.js

```diff
@@ -3,199 +3,129 @@ import PropTypes from 'prop-types';
   }
 
   /**
-   * Create a linear gradient using the received color and a color obtained adjusting the brightness
-   * of that color using props.gradientBrightness. The order is [adjusted color, received color];
-   * this order can be reversed using reverseDirection.
-   * @param {string} color of the item
-   * @param {boolean} reverseDirection used to reverse the order of the colors in the gradient.
+   * Create a linear gradient using the base color(calls getGradientColor) and a color obtained adjusting
+   * the brightness of that color using getGradientBrightness(). The default order of the colors is
+   * [brighter color, color]; this order can be reversed if getReverseDirection() is true.
+   * 
+   * By default, the background of an item uses a gradient, this method should be overriden if this behaviour is not wanted.
    * @returns {string} linear gradient
    */
-  getGradient(color, reverseDirection) {
+  getBackgroundGradient() {
     let colors = [
-      Color(color)
-        .lightness(this.props.gradientBrightness)
+      Color(this.getGradientColor())
+        .lightness(this.getGradientBrightness())
         .hexString(),
-      color
+      this.getGradientColor()
     ];
-    if (reverseDirection) {
+    if (this.getReverseDirection()) {
       colors.reverse();
     }
-    return (
-      'linear-gradient(' +
-      colors[0] +
-      ' ' +
-      (this.props.gradientStop === undefined ? '' : this.props.gradientStop + '%') +
-      ', ' +
-      colors[1] +
-      ')'
-    );
+
+    return ('linear-gradient(' + colors[0] + ' ' + this.getGradientStop() + '%, ' + colors[1] + ')');
   }
 
   /**
-   * If this.props.useGradient true, create gradient using props.color; otherwise return the color.
-   * @returns background style
+   * Returns the style of the item.
    */
-  getBackground() {
-    if (this.props.useGradient) {
-      return this.getGradient(this.props.color, this.props.reverseDirection);
-    } else {
-      return this.props.color;
+  getStyle() {
+    // subtract 10 because of the margin (see rct9k-items-inner class in style.css)
+    const itemHeight = this.props.itemHeight ? this.props.itemHeight - 10 : 'auto';
+
+    return {
+      ...this.props.style,
+      height: itemHeight,
+      background: this.getBackgroundGradient()
     }
   }
 
   /**
-   * Create border style: borderThickness, cornerRadius and borderColor.
-   * @returns border style
+   * Returns the css classes applied on the item.
    */
-  getBorder() {
-    let border = {
-      borderRadius: this.props.cornerRadius,
-      borderWidth: this.props.borderThickness,
-      borderStyle: this.props.borderThickness !== undefined ? TIMELINE_ITEM_BORDER_STYLE : '',
-      borderColor: this.props.borderColor
-    };
-    return border;
+  getClassName() {
+    let className = this.props.className;
+    if (this.props.item.glowOnHover) {
+      className += ' ' + ITEM_GLOW_CLS;
```

Comment:
1) externalize `getGlowOnHover()`

2) let's also have a class for the normal state. And put in it all the things that we had in the initial impl, but we removed. opacity = a subtle transparency; around alpha = .8, border color = try dark gray, border thickness = 1 or 2, radius = discrete rounded corners; around 3 px? Did I forget something?

Reply:
Done